generators/entity-server/templates/src/main/java/package/domain/Entity.java.ejs

```diff
@@ -285,10 +285,16 @@ public class <%= asEntity(entityClass) %> implements Serializable {
 }
 
 relationships.forEach((relationship, idx) => {
-    const otherEntityRelationshipName = relationships[idx].otherEntityRelationshipName;
+    const otherEntityRelationshipName =
```

Comment:
If there is a different nomenclature for relationship name, the rule should be applied where it is calculated?

Reply:
I think this nomenclature should only apply to the database mapping itself and this is the first time it is calculated

https://neo4j.com/docs/cypher-manual/current/syntax/naming/#_recommendations

Reply:
Otherwise it would be also used in the front end

Reply:
@Evertude I was talking to calculate it directly on function _prepareRelationshipForTemplates_, but @mashima can confirm if it is the correct function.

Reply:
@mshima @atomfrede If we re-calculate this relationship name here, this can be an issue due to the size of the relationship length (we are adding more 4 characters HAL_)?

Reply:
Hard to say, this is a neo4j exclusive feature and there is no neo4j test workflow to see if it works.
I know for sure that those won't be propagated to others generators and templates, and changing the main relationshipName will break others generators like frontend.
I will let neo4j experts to decide.